logstash-codec-protobuf.gemspec

```diff
@@ -20,7 +20,9 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
-  s.add_runtime_dependency 'ruby-protocol-buffers' # used by the compiled version of our protobuf definition. 
+  # s.add_runtime_dependency 'google-protobuf', '3.1'
+  s.add_runtime_dependency 'google-protobuf', '3.5.0.pre'
```

Comment:
since 3.5.1.2 is out, can we bump this to "~> 3.5" so we don't depend on a pre released gem?

Reply:
@jsvd tried that already and ran into issues, @jorgelbg will send you the details

Reply:
@jsvd The issue is that the `google-protobuf` library haven't released the java version for the 3.5.1.2 (https://rubygems.org/gems/google-protobuf/versions). The last version available is the 3.5.0.pre that we're using.

Reply:
I didn't notice :(
probably it makes sense in a near future to directly use java libraries for this, it seems the jruby part of this gem isn't being maintained

Reply:
That is one possibility, it would eliminate the advantage of having the native JRuby classes without the calling Java overhead. But, with all the changes on the java pipeline happening on master perhaps in the future would be plausible 👍.